src/dmd/parse.d

```diff
@@ -1515,6 +1515,37 @@ final class Parser(AST) : Lexer
         }
     }
 
+    //Indicates if a pack of TypeCtor, as allowed before a BasicType, can be read.
```

Comment:
I have no idea what a "pack" is. Need to:

1. give an example of what a "pack" is
2. give URL to spec on what is going on here

Reply:
Also rephrase the comment to use the Ddoc `returns: ... explanation...` style.

Reply:
Okay. By pack I meant a "sequence of".

Reply:
function removed, as suggested by Mike F.